docs/usage.md

```diff
@@ -69,119 +115,360 @@ cat ./compilation.json
   and in most cases, _System Integrity Protection_ needs to be turned off.
   See the [README](/README.md#mac-os-x) for details.
 
-## Step 2: Analyze your code <a name="step-2"></a>
-Once the build is logged successfully (and the `compilation.json`) was created,
-you can analyze your project.
+## Step 2: Analyze your code<a name="step-2"></a>
+Once the build is logged successfully and the `compile_commands.json` was
+created, you can analyze your project.
 
-1. Run the analysis:
+### Run the analysis<a name="run-the-analysis"></a>
 ```sh
-CodeChecker analyze compilation.json -o ./reports
+CodeChecker analyze ./compile_commands.json --enable-all --output ./reports
 ```
-2. View the analysis results in the command line
+
+However the compilation of the project (here the example program) is performed
+by `gcc`, the CodeChecker uses `clang` to analyze sources of the project.
+During analysis, CodeChecker compiles/analyses sources again. The analysis
+process generally uses more time. It would be good to mention here that if you
+want to speed up analysis specify a higher value for the `--jobs` option.
+
+In the above command the `--enable-all` means that a subset of checker
+are run. `enable-all` chooses a predefined "group" of checkers. The further
+info on available checkers use
```

Comment:
Don't mention the `--enable` option is this user guide because it is not recommended option. It is only for debug purposes.

Reply:
Changed to sensitive group.